vllm/engine/arg_utils.py

```diff
@@ -432,12 +432,25 @@ def from_cli_args(cls, args: argparse.Namespace) -> 'EngineArgs':
     def create_engine_config(self, ) -> EngineConfig:
         device_config = DeviceConfig(self.device)
         model_config = ModelConfig(
-            self.model, self.tokenizer, self.tokenizer_mode,
-            self.trust_remote_code, self.download_dir, self.load_format,
-            self.dtype, self.seed, self.revision, self.code_revision,
-            self.tokenizer_revision, self.max_model_len, self.quantization,
-            self.quantization_param_path, self.enforce_eager,
-            self.max_context_len_to_capture, self.max_logprobs)
+            self.model,
+            self.tokenizer,
+            self.tokenizer_mode,
+            self.trust_remote_code,
+            self.download_dir,
+            self.load_format,
+            self.dtype,
+            self.seed,
+            self.revision,
+            self.code_revision,
+            self.tokenizer_revision,
+            self.max_model_len,
+            self.quantization,
+            self.quantization_param_path,
+            # UPSTREAM SYNC: keep sparsity argument
+            self.sparsity,
+            self.enforce_eager,
+            self.max_context_len_to_capture,
+            self.max_logprobs)
```

Comment:
should we be using names for some/most of these arguments?

Reply:
i do want to, but that would further diverge from the upstream so i didn't go for it

Reply:
lets do an upstream PR for this